clio-client/src/main/resources/reference.conf

```diff
@@ -19,6 +19,7 @@ clio {
   greenteam {
     email = "greenteam@broadinstitute.org"
   }
+  service-account-json = null
```

Comment:
Could this be nested under `client`? Same with `greenteam`, though that's been around for a bit.